src/theme/scss/chip/chip.scss

```diff
@@ -0,0 +1,6 @@
+.mdl-chip {
+  color: black;
+  &__action {
+    color: inherit;
+  }
+}
```

Comment:
newline at the end of the file
add pre-commit rule  `lint` on the package.json

Reply:
✔️